### ISSUE TITLE
Builtins and standard library

### ISSUE THREAD

**Issue by metaphori** [scafi/scafi#11]
Originally reported by: **Roberto Casadei (Bitbucket: [metaphori](https://bitbucket.org/metaphori), GitHub: [metaphori](https://github.com/metaphori))**

----------------------------------------

Currently, the standard library is implemented in a module, `stdlib`, which is distinguished from the `core` module. However, the `core` module does provide some library support (cf., `RichLanguage` component and its `Builtins`).

The idea behind a separated `stdlib` module lies in the fact of keeping the `core` minimal and with few dependencies.

Basically, this task is about considering, from the point of view of design:

- whether to maintain this arrangement; or
- whether to move the `stdlib` to `core`; or
- whether to move builtins from `core` to `stdlib`..

----------------------------------------
- Bitbucket: https://bitbucket.org/scafiteam/scafi/issue/6
